addons/web/static/src/scss/utilities_custom_backend.scss

```diff
@@ -22,7 +22,16 @@ $utilities: map-merge(
 
         // Disable 'bg-X' classes generation.
         // See bootstrap_review_backend.scss
-        "background-color": null
+        "background-color": (
+            property: background-color,
+            class: bg,
+            local-vars: (
+                "bg-opacity": inherit
```

Comment:
Initially I was puzzled by `inherit` here, but after testing locally I now understand your struggles to set the right CSS variable's value ^^

I do propose to don't touch this file and generate the classes using `bootstrap_review_backend.scss` directly.

Let's keep in mind that the compiled result should be something like this
```css
.bg-transparent {
    --background-color: RGBA(0, 0, 0, var(--bg-opacity, 0));
    background-color: var(--background-color) !important;
}
```

Reply:
I literally replicate the expected output.

I tried to use the `o-print-color` mixin but it would set the opacity to 1 and extending `bg-black` doesn't work either.

Is this what you were suggesting?